spec/lib/miq_helper_spec.rb

```diff
@@ -0,0 +1,68 @@
+# This tests are done in a way that assume the Rails environment for Miq are
+# loaded, and that Rails is present to do the comparisons.  Tests that confirm
+# that this works when rails is not present us a subprocess to run the
+# `lib/miq.rb` file in isolation.
+#
+# If tests are not passing, check to see if the spec/spec_helper.rb is being
+# loaded properly and initailizing the Vmdb::Application.
+
+require 'miq_helper'
```

Comment:
If you name it correctly, this is not needed since the constant lookup on line 11 would autoload it.